deploy/registration-service/registration-service.yaml

```diff
@@ -177,24 +163,48 @@ objects:
       tls:
         termination: edge
       wildcardPolicy: None
+
+  # service associated with the registration service route
   - kind: Service
     apiVersion: v1
     metadata:
-      name: api
+      name: registration-service
```

Comment:
I reordered the Routes and Services and added a few comments, but unfortunately, the `diff` doesn't clearly shows how the items were reordered :/